mappings/net/minecraft/text/ScoreText.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_2578 net/minecraft/text/ScoreText
 	FIELD field_11785 objective Ljava/lang/String;
 	FIELD field_11786 selector Lnet/minecraft/class_2300;
 	FIELD field_11787 name Ljava/lang/String;
+	FIELD field_33290 EVERYTHING Ljava/lang/String;
```

Comment:
This field doesn't seem to be a wildcard. `ScoreText.parse` compares the input with this field to determine if it should use the sender name.

Reply:
```suggestion
	FIELD field_33290 SENDER_PLACEHOLDER Ljava/lang/String;
```

Reply:
Ahh thanks